settings.py

```diff
@@ -1069,6 +1069,8 @@ def _pkce_required(client_id):
     },
 }
 
+ES_CASE_SEARCH_INDEX_NAME = "case_search_2018-05-29"
```

Comment:
I think we can get rid of the setting in this file entirely. It's only needed when doing a reindex so that you can override the default.